packages/flutter/lib/src/material/chip.dart

```diff
@@ -1784,23 +1784,26 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
     if (!hasDeleteButton) {
       return null;
     }
-    return _wrapWithTooltip(
-      widget.deleteButtonTooltipMessage ?? MaterialLocalizations.of(context)?.deleteButtonTooltip,
-      widget.onDeleted,
-      GestureDetector(
-        key: deleteIconKey,
-        behavior: HitTestBehavior.opaque,
-        onTap: widget.isEnabled
-            ? () {
-                Feedback.forTap(context);
-                widget.onDeleted();
-            }
-            : null,
-        child: IconTheme(
-          data: theme.iconTheme.copyWith(
-            color: widget.deleteIconColor ?? chipTheme.deleteIconColor,
+    return Semantics(
```

Comment:
I wonder if this change should be moved inside the `_wrapWithTooltip` helper

Reply:
Good question. I chose not to because there is already a top level Semantics node for the whole chip, and the only other use case of _wrapTooltip is for the onPressed, and that tooltip message gets read from the top level Semantics node